deploy/rosa-console-branding/rosa-branding.console.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: operator.openshift.io/v1
+kind: Console
+metadata:
+  name: cluster
+spec:
+  managementState: Managed
+  authentication:
+    # the logout redirect must be a valid URI ethat starts with http(s)
+    # valid examples: http://localhost:8080/logout.html, https://www.redhat.com/log/out/
+    logoutRedirect: "https://www.redhat.com/i/should/logout"
+  customization:
+    customProductName: Red Hat OpenShift Service on AWS
+    customLogoFile:
+      name: rosa-brand-logo
+      key: rosa-brand-logo.svg
+    # the documentation base must be a valid URI that starts with http(s) AND it must have a trailing slash
+    # the trailing slash is required because additional values will be appended
+    # valid examples: https://localhost:9999/docs/, http://192.168.1.5:3000/doc/location/
+    documentationBaseURL: "https://docs.openshift.com/rosa/"
```

Comment:
Confirm this is the final documentation base and add to jira description.

Reply:
Confirmed in JIRA